shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -70,27 +104,17 @@ fn test_cluster() {
 
 #[test]
 #[serial]
-fn test_source_tls_and_single_tls() {
+#[cfg(feature = "alpha-transforms")]
+fn test_source_tls_and_cluster_tls() {
     test_helpers::cert::generate_cassandra_test_certs();
-    let _compose = DockerCompose::new("example-configs/cassandra-tls/docker-compose.yml");
+    let _compose = DockerCompose::new("example-configs/cassandra-cluster-tls/docker-compose.yml");
 
     let shotover_manager =
-        ShotoverManager::from_topology_file("example-configs/cassandra-tls/topology.yaml");
+        ShotoverManager::from_topology_file("example-configs/cassandra-cluster-tls/topology.yaml");
 
     let ca_cert = "example-configs/cassandra-tls/certs/localhost_CA.crt";
 
-    {
-        // Run a quick test straight to Cassandra to check our assumptions that Shotover and Cassandra TLS are behaving exactly the same
```

Comment:
Could we include this direct check for the cluster test as well?
It would help prevent the case where the test cluster is not setup for TLS properly and shotover is not implementing TLS properly.